pkg/kudoctl/util/repo/repo_operator.go

```diff
@@ -75,5 +84,39 @@ func (c *Client) DownloadIndexFile() (*IndexFile, error) {
 	}
 
 	indexFile, err := ParseIndexFile(indexBytes)
+	//TODO (kensipe): err handling such that error in includes are ignored (but not root)
+	//TODO (kensipe): track which includes have happened so there are no repeats
+	for _, include := range indexFile.Includes {
+		iURL, err := url.Parse(include)
+		if err != nil {
+			clog.Printf("Unable to parse include url for %s", include)
+		}
+		nextIndex, err := c.downloadIndexFile(indexFile, iURL)
+		if err != nil {
+			return nil, err
```

Comment:
👍 I think this is important, we definitely should not silently (or with a warning) skip a repository entry here.